impeller/tessellator/c/tessellator.h

```diff
@@ -42,7 +42,8 @@ IMPELLER_API void Close(PathBuilder* builder);
 
 IMPELLER_API struct Vertices* Tessellate(PathBuilder* builder,
                                          int fill_type,
-                                         Scalar scale,
+                                         Scalar tolerance,
+                                         // Unused, remaining for ABI stability.
```

Comment:
Can we clean this up at all?

Reply:
vector_graphics is potentially using these parameters and would crash without them. Although I don't think we actually have anything using them right now - I'll just drop them for now and we can clean up vg separately.